packages/doenetml-worker-rust/lib-doenetml-core/src/core/state_var_interfaces/boolean_state_var_interfaces.rs

```diff
@@ -21,9 +21,12 @@ use super::util::{create_dependency_instruction_from_extend_source, string_to_bo
 /// then propagate the `used_default` attribute of the essential state variable.
 #[derive(Debug, Default)]
 pub struct GeneralBooleanStateVarInterface {
+    is_primary_state_variable: bool,
```

Comment:
Can you add docstrings explaining what these are? Maybe even with sample code?

Reply:
Added docstring, but didn't add sample code. Simpler now, though, without primary state variable